app/javascript/segment_graph_builder.js

```diff
@@ -0,0 +1,73 @@
+import {build_segment_graph} from "graphs/segment_duration_graph.js"
+import {Spinner} from 'spin.js'
+
+$(function() {
+  const graphHolders = document.getElementsByClassName('segment-graph-holder')
+  if(graphHolders.length === 0) {
+    return
+  }
+
+  let spinners = []
```

Comment:
This should probably be `const` as well since we don't want to change the actual value at all.

Reply:
Wow I honestly did not understand `const` until this prompted me to go look it up, changed!